sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_generated/v7_3_preview/models/_key_vault_client_enums.py

```diff
@@ -0,0 +1,124 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from enum import Enum, EnumMeta
+from six import with_metaclass
+
+class _CaseInsensitiveEnumMeta(EnumMeta):
+    def __getitem__(self, name):
+        return super().__getitem__(name.upper())
+
+    def __getattr__(cls, name):
+        """Return the enum member matching `name`
+        We use __getattr__ instead of descriptors or inserting into the enum
+        class' __dict__ in order to support `name` and `value` being both
+        properties for enum members (which live in the class' __dict__) and
+        enum members themselves.
+        """
+        try:
+            return cls._member_map_[name.upper()]
+        except KeyError:
+            raise AttributeError(name)
+
+
+class DataAction(with_metaclass(_CaseInsensitiveEnumMeta, str, Enum)):
```

Comment:
Looks like the public KeyVaultDataAction enum needs an update. This has new values in addition to the ones added to 7.2.

Reply:
I was planning on updating the enum in a separate PR, but it's a small enough change that I may as well do it here